packages/sku/config/webpack/utils/index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   ...loaders,
   resolvePackage,
   TYPESCRIPT: /\.(ts|tsx)$/,
-  JAVASCRIPT: /\.js$/,
+  JAVASCRIPT: /\.(js|cjs|mjs)$/,
```

Comment:
Please use the extensions defined in `eslint-config-seek`.

We should have a single source of truth to avoid such situations in the future. Case in point: this doesn't include `jsx`, which is supported by our tooling ([1], [2], [3], [4]).

[1]: https://github.com/seek-oss/skuba/blob/2c640b6aeffff78f76ec7232b11fffca35bebc90/.eslintrc.js#L7
[2]: https://github.com/seek-oss/sku/blob/51186f39b0855c5c8b011810216a3d308b78a82b/packages/sku/lib/lint.js#L1-L4
[3]: https://github.com/seek-oss/sku/blob/721990812bb10d5c5879d19e15af29b652c5bbf7/packages/sku/lib/runPrettier.js#L7
[4]: https://github.com/seek-oss/eslint-config-skuba/blob/579a1fd97b25bf2023c5943ab7fca7078ca7c88f/index.js#L2

Reply:
I thought we were just going to get this out and then gold plate it later. I'm honestly unsure if we want to couple this config to the eslint config, not because I don't want to have a single source of truth, but more so that the list from the config is quite load bearing now, and I'm not sure if it's the best place for the source of truth the live.

Reply:
That being said, I don't see a reason not to include `jsx`. We'll use the extensions for now.